### ISSUE TITLE
Overall discussion for Tip 7

### ISSUE THREAD

**Issue by SiminaB** [Benjamin-Lee/deep-rules#248]
This is to discuss outstanding issues for Tip 7: Choose an appropriate data representation and neural network
architecture.
https://github.com/Benjamin-Lee/deep-rules/blob/master/content/09.overfitting.md

**Comment by SiminaB** [comment]
I'm not super sure about the figure here. It does a good job showing overfitting, but in general analysts are taught to be super cautious when extrapolating models outside of the range of the training data, so I wouldn't continue drawing the linear regression outside that range either.

**Comment by Benjamin-Lee** [comment]
I think you're right; it belongs in the "know your data" tip. It does inflate accuracy, but it's not due to memorization.
